client/blocks/upsell-nudge/style.scss

```diff
@@ -0,0 +1,52 @@
+.upsell-nudge {
+	border-left: 8px solid var( --color-accent );
+	background-color: var( --color-neutral-80 );
+	color: var( --color-text-inverted );
+	display: flex;
+	align-items: center;
+	margin-bottom: 1.5em;
+	padding: 0.5em 1.5em;
+
+	a {
+		color: var( --color-text-inverted );
+	}
+
+	&__content {
```

Comment:
Please don't compose class names with the `&` operator like this. The `.upsell-nudge__content` class name is then impossible to find by grep.

Reply:
Ah, thanks for the info! That's a good point about not being able to search. I'd seen this done with Customer Home so I assumed it was OK. At some point I'll go back and refactor Customer Home's CSS, too.

Reply:
Oops, I admit I used the same pattern on some styles for the hosting section:

https://github.com/Automattic/wp-calypso/blob/61ee4faffdc46799dda6fe2f38ea81ea818798ca/client/my-sites/hosting/style.scss#L17

I'll try to refactor them too.